std/range/package.d

```diff
@@ -10195,13 +10195,18 @@ in
             auto result = adds(start, range.length, overflow);
         else static if (isSigned!Enumerator)
         {
-            Largest!(Enumerator, Signed!LengthType) signedLength;
+            alias signed_t = Largest!(Enumerator, Signed!LengthType);
```

Comment:
I'm not sure it's worth giving this type a name. But if it is, then it's worth replacing the `typeof(signedLength)` below with it.